src/routes/api/index.js

```diff
@@ -2,13 +2,18 @@ import swaggerJsdoc from 'swagger-jsdoc';
 import swaggerUi from 'swagger-ui-express';
 import { Router } from 'express';
 import swaggerOptions from 'utils/api-specifications';
-import welcomeRoute from './welcome';
+import isLoggedIn from '../../utils/is-logged-in';
 import userRoutes from './user.routes';
 
 const router = Router();
 const swaggerDoc = swaggerJsdoc(swaggerOptions);
 router.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerDoc));
-router.use('/', welcomeRoute);
+router.get('/', isLoggedIn);
```

Comment:
Why was this route '/' removed? yet it is the default route that displays the welcome message

Reply:
<img width="1440" alt="Screen Shot 2020-02-17 at 11 58 51" src="https://user-images.githubusercontent.com/52036624/74643143-e9e33e80-517c-11ea-8f7c-b8ceb8ddbe18.png">


@AbdoulNuru, as you can see in the image our https://octopus-bn-backend-staging.herokuapp.com is not returning any status or anything meaningful and when I was trying to create an SSL certificate, is was giving an error so to fix it I had just to implement something and return a given status. 


but you can test the main route by navigating to localhost:3000 you will still see the welcome message

Reply:
Yes that welcome route was removed by someone, but the TTL changed it back to what it was before(you can check from develop branch), so my worry is that now you are removing it again.

Reply:
done  please navigate to `localhost:3000 `